e2e-tests/src/test/committee_split.rs

```diff
@@ -0,0 +1,151 @@
+use std::{thread::sleep, time::Duration};
+
+use aleph_client::{
+    pallets::{elections::ElectionsSudoApi, session::SessionApi},
+    primitives::CommitteeSeats,
+    utility::BlocksApi,
+    waiting::{BlockStatus, WaitingExt},
+    AccountId, AsConnection, Pair, SignedConnection, TxStatus,
+};
+use log::info;
+
+use crate::{
+    accounts::{get_validator_seed, get_validators_keys, NodeKeys},
+    config::setup_test,
+    rewards::set_invalid_keys_for_validator,
+};
+
+/// time needed for 5 out of 7 block producers to do 3 sessions.
+const SLEEP_DURATION: Duration = Duration::from_secs(126);
```

Comment:
This is suspiciously specific. We cannot really rely on sessions taking a constant time, so it either requires some criteria checking or a time with a margin of error (if applicable).

Reply:
I would avoid weird-looking numbers at least – make it 130 and add an "about" or "at least" to the comment to make things clearer please.

Reply:
done